daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -113,6 +143,33 @@ Connection: close 
  }
 ```
 
+## Headers
+
+As part of the start HTTP request, the caller can optionally include one or more `dapr-workflow-metadata` HTTP request headers. The format of the header value is a list of `{key}={value}` values, similar to the format for HTTP cookie request headers. These key/value pairs are saved in the workflow instance’s metadata and can be made available for search (in cases where the workflow implementation supports this kind of search).
```

Comment:
Can we remove this whole `## Headers` section? It's not correct. We have no such feature today.

Reply:
That is what I figured but kept it around, will remove it.

Reply:
Updated